nipype/interfaces/freesurfer/petsurfer.py

```diff
@@ -0,0 +1,651 @@
+    )
+
+    mg_ref_cerebral_wm = traits.Bool(
+        argstr="--mg-ref-cerebral-wm", desc=" set MG RefIds to 2 and 41"
+    )       
+
+    mg_ref_lobes_wm = traits.Bool(
+        argstr="--mg-ref-lobes-wm", desc="set MG RefIds to those for lobes when using wm subseg"
+    )      
+
+    mgx = traits.Float(
+        argstr="--mgx %f",
+        desc="gmxthresh : GLM-based Mueller-Gaertner PVC, gmxthresh is min gm pvf bet 0 and 1"
+    ) 
+
+    km_ref = traits.List(
+        argstr="--km-ref %s...", desc="RefId1 RefId2 ... : compute reference TAC for KM as mean of given RefIds"
+    )
+
+    km_hb = traits.List(
+        argstr="--km-hb %s...", desc="RefId1 RefId2 ... : compute HiBinding TAC for KM as mean of given RefIds"
+    ) 
+
+    ss = traits.Tuple(
+        traits.Float,
+        traits.Float,
+        traits.Float,
+        argstr="--ss %f %f %f",
+        desc="bpc scale dcf : steady-state analysis spec blood plasma concentration, unit scale and decay correction factor. You must also spec --km-ref. Turns off rescaling"
+    ) 
+
+    X = traits.Bool(
+        argstr="--X", desc="save X matrix in matlab4 format as X.mat (it will be big)"
+    ) 
+
+    y = traits.Bool(
+        argstr="--y", desc="save y matrix in matlab4 format as y.mat"
+    )
+
+    beta = traits.Bool(
+        argstr="--beta", desc="save beta matrix in matlab4 format as beta.mat"
+    )
+    
+    X0 = traits.Bool(
+        argstr="--X0", desc="save X0 matrix in matlab4 format as X0.mat (it will be big)"
+    )
+
+    save_input = traits.Bool(
+        argstr="--save-input", desc="saves rescaled input as input.rescaled.nii.gz"
+    )
+
+    save_eres = traits.Bool(
+        argstr="--save-eres", desc="saves residual error"
+    )
+
+    save_yhat = traits.Bool(
+        argstr="--save-yhat", desc="save signal estimate (yhat) smoothed with the PSF"
+    )
+
+    save_yhat_with_noise = traits.Tuple(
+        traits.Int,
+        traits.Int,
+        argstr="--ss %i %i",
+        desc="seed nreps : saves yhat with noise, seed < 0 for TOD"
+    ) 
+    
+    traits.Bool(
+        argstr="--save-yhat-with-noise", desc="save signal estimate (yhat) with noise"
+    ) 
+
+    save_yhat_full_fov = traits.Bool(
+        argstr="--save_yhat_full_fov", desc="save signal estimate (yhat)"
+    )
+
+    save_yhat0 = traits.Bool(
+        argstr="--save_yhat0", desc="save signal estimate (yhat)"
+    )
+
+    opt = traits.Int(
+        argstr="--opt %i",
+        desc="opt : optimization schema for applying adaptive GTM"
+    )  
+
+    opt_tol = traits.Tuple(
+        traits.Float,
+        traits.Float,
+        traits.Float,
+        argstr="--opt-tol %f %f %f",
+        desc=""
+    ) 
+
+    opt_brain = traits.Bool(
+        argstr="--opt-brain", desc="apply adaptive GTM"
+    ) 
+
+    opt_seg_merge = traits.Bool(
+        argstr="--opt-seg-merge", desc="optimal schema for merging ROIs when applying adaptive GTM"
+    ) 
+
+    threads = opt = traits.Int(
```

Comment:
```suggestion
    threads = traits.Int(
```

Also, should this be a `FSCommandOpenMP` interface?

https://github.com/nipy/nipype/blob/79132b519e3570503712920aff3f32b5f9d2d097/nipype/interfaces/freesurfer/base.py#L228-L264

If not, you still might want to change it to `num_threads` for consistency with other interfaces.